update introduction prose (#322)
i0natan/nodebestpractices: README.md

<br/>

> # Welcome! First, three things you should know:

Reviewer: Yup, we can keep it as is — "3 Things You Ought To Know First".

> **1. The content presented here is a curation of the best Node.js articles -** this repository is a summary and curation of the top-ranked content on Node.js best practices, as well as content written here by collaborators

Reviewer: nit: The idea of 'curation' is repeated in the bullet and its description. The original text, in spirit, seemed more 'DRY'.

```suggestion
**1. You are, in fact, reading dozens of the best Node.js articles -** this repository is a summary and curation of the top-ranked content on Node.js best practices, as well as content written by collaborators
```

**2. It is the largest compilation, and it is growing every week -** currently, more than 70 best practices, style guides, and architectural tips are presented. New issues and pull requests are created every day to keep this live book updated. We'd love to see you contributing here, whether that is fixing code mistakes, helping with translations, or suggesting brilliant new ideas. See our [writing guidelines here](/.operations/writing-guidelines.md)

> **3. Most best practices have additional info -** most best practices include a **🔗Read More** link that expands on the practice with code examples, quotes from selected blogs and more information

Reviewer: nit: 'practices' occurs thrice; replacing one occurrence with 'bullets' seems to flow better.

```suggestion
**3. Most best practices have additional info -** most bullets include a **🔗Read More** link that expands on the practice with code examples, quotes from selected blogs and more information
```
